addons/mail/static/src/models/thread/thread.js

```diff
@@ -1632,6 +1632,15 @@ function factory(dependencies) {
             this.unregisterOtherMemberTypingMember(partner);
         }
 
+        /**
+         * Disable composer input and buttons based on the chat sessions.
+         *
+         * @private
+         * */
+        disableComposer() {
+            this.update({ isDisabled: true });
+        }
+
```

Comment:
As per suggestion above this should be replaced by a compute that depends on livechat active